Agreements/Internal/Gal_Ant.txt

```diff
@@ -1,3 +1,4 @@
 COPX:EGPK:*:GIRVA:*:*:Galloway:Antrim:10000:*:^GIRVA
```

Comment:
```suggestion
```

This agreement has been replaced - if we had two COPX with the same fix then the one below wouldn't be matched